pkg/tfgen/docs.go

```diff
@@ -796,6 +796,14 @@ func (p *tfMarkdownParser) parseAttributesReferenceSection(subsection []string)
 }
 
 func (p *tfMarkdownParser) parseImports(subsection []string) {
+	// check for import overwrites
+	importDetails := p.info.GetImportDetails()
+	if importDetails != "" {
+		var details []string
+		details = append(details, importDetails)
```

Comment:
Not sure why this is necessary. Could you explain with a comment or remove if unnecessary?

Reply:
This is currently a []string so we need to build that list of []string

Reply:
So why not:

```go
details := []string{
  importDetails,
}
```

Reply:
Yeah, we really don't need this, as we're just writing to `entityDocs.Import` which is a plain string